client/src/main/com/sinch/sdk/domains/voice/adapters/converters/ApplicationsDtoConverter.java

```diff
@@ -69,7 +69,7 @@ public static NumberInformation convert(GetQueryNumberDto dto) {
         .build();
   }
 
-  public static UpdateNumbersDto convert(ApplicationsUpdateNumbersRequestParameters client) {
+  public static UpdateNumbersDto convert(ApplicationsAssignNumbersRequestParameters client) {
```

Comment:
The Dto is still `Update` - is this intended?

Reply:
Yes, all DTO related objects/files are generated and coming from OAS.
OAS operation is named "Update" so stay named "Update"